Modules/Office365DSC/DSCResources/MSFT_SPOAccessControlSettings/MSFT_SPOAccessControlSettings.schema.mof

```diff
@@ -0,0 +1,18 @@
+[ClassVersion("1.0.0.0"), FriendlyName("SPOAccessControlSettings")]
+class MSFT_SPOAccessControlSettings : OMI_BaseResource
+{
```

Comment:
The descriptions for the params now automatically drive the wiki documentation. Please make sure each sentence finishes with a ‘.’